packages/legend-query-builder/src/stores/LegendQueryStorage.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum LEGEND_QUERY_SETTINGS_KEY {
+  EDITOR_SHOW_POST_FILTER = 'editor.showPostFilter',
```

Comment:
the keys could be permanent so let's be a little more mindful with it:
--> `QUERY_BUILDER_SHOW_POST_FILER` = 'query-builder.showPostFilter`